blog/2023-06-15-paralus-and-Kamaji-integration/index.md

```diff
@@ -0,0 +1,76 @@
+ 
+## Solution Showcase: Kamaji and Paralus Integration
+By combining Kamaji and Paralus, organizations can achieve a powerful solution for managing multiple Kubernetes clusters and implementing advanced RBAC practices. 
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/JWA2LwZazM0" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe> 
+
+<br/>
+
+#### The following takes are available thanks to Kamaji:
+- **Fast provisioning of Control Planes:** running as Pods, the onboard process is blazing fast, allowing the creation of a fully compliant CNCF Kubernetes Control Plane in less than 30 seconds.
+- **Constant reconciliation of Control Planes:** ensuring high availability and the desired state, Kamaji constantly checks the current state of your definitions with the desired one, removing any operational burden in regards to bootstrap and maintenance.
+- **Single pane of glass for multi-cluster management:** managing a fleet of control planes in a declarative, GitOps, and Cloud-Native way allows your organization to tame the multi-cluster challenges, allowing a self-service approach according to the latest Platform Engineering needs.
+ 
+
+#### Thanks to Paralus, we can achieve the following outcomes:
+- **Secure Access Federation:** Paralus acts as the entry point for accessing all managed clusters, ensuring secure authentication and authorization across the board. It enforces consistent access controls and policies, reducing the risk of unauthorized access.
+- **RBAC Simplification:** Paralus provides a centralized interface for managing RBAC across all clusters, enabling administrators to define and enforce role-based access at scale. This streamlines the process of granting permissions and ensures consistent authorization policies for developers.
+- **Audit Logging and Compliance: **Paralus captures detailed audit logs of user activities, allowing organizations to meet compliance requirements and investigate any security incidents or policy violations efficiently.
+
+## Here's how the integration works
+ 
+<img src="/img/docs/paralus-kamaji-blog.png" alt="paralus and kamaji integration" height="90%" width="90%"/>  
+<br/>
+
+- **Installing Paralus on the management cluster:** it will be used to expose the access and control the users' RBAC for the workloads clusters.
+- **Provisioning Control Planes on Kamaji:** once ready, a valid kubeconfig will be generated and can be used by the Platform administrators.
+- **Install the Paralus agent on each Tenant Control Plane:** for each cluster and their kubeconfig, the Paralus agent must be installed, waiting for their registration on the Paralus dashboard.
```

Comment:
this can just be `for each cluster, the paralus agent must be installed ..... `

not sure what installing paralus agent on each cluster and their kubeconfig means

Reply:
modifying this